frontend/src/components/Profile/profile.module.css

```diff
@@ -0,0 +1,154 @@
+.profile-page {
+    display: grid;
+    height: 100%;
+    width: 100%;
+    background-color: rgb(254, 239, 221);
```

Comment:
i forget, have we defined css variables yet for our common colors?

Reply:
We haven't defined css variables for our colors. Should I create them?